src/transaction/log_replication_atomic.cpp

```diff
@@ -83,17 +83,11 @@ namespace cublog
 	    read_and_redo_record<LOG_REC_COMPENSATE> (thread_entry, header, m_redo_lsa);
 	    break;
 	  case LOG_DBEXTERN_REDO_DATA:
-	  {
-	    m_redo_context.m_reader.advance_when_does_not_fit (sizeof (LOG_REC_DBOUT_REDO));
-	    const LOG_REC_DBOUT_REDO dbout_redo =
-		    m_redo_context.m_reader.reinterpret_copy_and_add_align<LOG_REC_DBOUT_REDO> ();
-	    log_rcv rcv;
-	    rcv.length = dbout_redo.length;
-
-	    log_rv_redo_record (&thread_entry, m_redo_context.m_reader, RV_fun[dbout_redo.rcvindex].redofun, &rcv,
-				&m_redo_lsa, 0, nullptr, m_redo_context.m_redo_zip);
+	    /* Recovery redo for RVDK_NEWVOL and RVDK_EXPAND_VOLUME will be called here,
```

Comment:
If you want to skip all redo replication for LOG_DBEXTERN_REDO_DATA. I think it'd be better to leave a comment where `LOG_DBEXTERN_REDO_DATA` is defined: will be not replicated. Although now there are only 3 recv-idx and they are not needed for replication, someone will possibly add a recvindex. Just a suggestion.

Reply:
By the way,  is it OK to skip `RVLOG_OUTSIDE_LOGICAL_REDO_NOOP` too?

Reply:
> If you want to skip all redo replication for LOG_DBEXTERN_REDO_DATA. I think it'd be better to leave a comment where LOG_DBEXTERN_REDO_DATA is defined: will be not replicated. Although now there are only 3 recv-idx and they are not needed for replication, someone will possibly add a recvindex. Just a suggestion.

Thanks for suggestion. I think it is better. 

> By the way, is it OK to skip RVLOG_OUTSIDE_LOGICAL_REDO_NOOP too?

recovery redo function for `RVLOG_OUTSIDE_LOGICAL_REDO_NOOP` (log_rv_outside_noop_redo ()) does nothing but returns NO_ERROR during replication. So I skipped this.. but do you know something about this recovery index ?

Reply:
No, I don't know too.

Reply:
If so, it seems to be better to block only the rcvindex that does fileio inside. (RVDK_NEWVOL, RVDK_EXPAND_VOLUME).